playbooks/nfd_operator_undeploy.yml

```diff
@@ -4,4 +4,4 @@
   connection: local
   gather_facts: true
   roles:
-    - role: nfd_undeploy
+    - role: nfd_operator_undeploy_from_operatorhub
```

Comment:
role is called nfd_operator_undeploy

Reply:
yes, I missed part of the renaming, solve with the next push